thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -35,6 +34,7 @@
 import org.joda.time.DateTime;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import static com.linkedin.thirdeye.anomaly.utils.ThirdeyeMetricUtil.*;
```

Comment:
Use "import com.linkedin.thirdeye.anomaly.utils.ThirdeyeMetricUtil;" like the other files does?